open_src/influx/influxql/yyParser.go

```diff
@@ -159,3 +159,8 @@ type cqSamplePolicyInfo struct {
 	ResampleEvery time.Duration
 	ResampleFor   time.Duration
 }
+type fieldList struct {
```

Comment:
why named it as fieldList? it is not a list. It is recommended to rename it, lick fieldOptions or others.